backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

```diff
@@ -8,6 +8,13 @@ toc::[]
 
 As a cluster administrator, you install the OpenShift API for Data Protection (OADP) by installing the OADP Operator. The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
 
+[IMPORTANT]
+====
+Use OADP 1.0._x_ with MTC 1.7._x_.
```

Comment:
I don't think these are suppose to be the warning messages; the OADP operator is installed as part of the MTC installation automatically and is limited to a specifc range (as @rayfordj mentioned in the jira ticket).

We should tell the customer that the next OADP 1.0.x releases starting from 1.0.4 are only supported when used as a dependency of MTC 1.7.z, and not as a standalone operator.